activerecord/test/cases/readonly_test.rb

```diff
@@ -48,6 +48,13 @@ def test_cant_touch_readonly_record
     assert_equal "Developer is marked as readonly", e.message
   end
 
+  def test_cant_touch_readonly_column
+    person = Person.find(1)
+
+    e = assert_raise(ActiveRecord::ActiveRecordError) { person.touch(:born_at)  }
```

Comment:
Yes, I think so, why not? How about to made this as a followup to this PR?

Reply:
Yeah, I just wanted to bring this up in case if core folks have opinion on that. We may need to come up with an example   or an least an argument why it might be useful to have a separate exception and rescue from it or maybe it's useful  because it conveys additional information to the exception reporting tooling. Otherwise it's very close to being considered a cosmetic change 🙃 
I would love to participate in a discussion in a separate PR if you feel like implementing the change :)